src/api/types.d.ts

```diff
@@ -0,0 +1,52 @@
+import { Timestamp } from 'firebase/firestore';
```

Comment:
Does really firebase require to define global types in js project?

Reply:
No, Firebase does not, but I do. Nothing else uses those types except for your IDE. It is not changing anything in the codebase. It just makes development waaaaaay easier. It is not even possible to include typescript file into JavaScript file. It does not get compiled or anything.